packages/flutter_tools/lib/src/flutter_features.dart

```diff
@@ -47,6 +47,14 @@ class FlutterFeatureFlags implements FeatureFlags {
   @override
   bool get isFlutterWebWasmEnabled => isEnabled(flutterWebWasm);
 
+  @override
+  bool get isCliAnimationEnabled {
+    if (_platform.environment['TERM'] == 'dumb') {
```

Comment:
never heard of this setting. I wonder, does anything modern still use this, or will this only affect Flutter developers on an old 70s teletype?

Reply:
I'm not sure, but from what I can tell it's what some software uses to indicate that they don't want any fancy shenanigans happening on the terminal, so, might as well, right...